src/pudl/glue/ferc1_eia.py

```diff
@@ -36,17 +36,20 @@
 
 import pudl
 from pudl.helpers import get_logger
+from pudl.metadata.fields import apply_pudl_dtypes
+from pudl.transform.classes import StringNormalization, normalize_strings
 from pudl.transform.ferc1 import Ferc1AbstractTableTransformer, Ferc1TableId
 
 logger = get_logger(__name__)
 
 PUDL_ID_MAP_XLSX = importlib.resources.open_binary(
     "pudl.package_data.glue", "pudl_id_mapping.xlsx"
 )
+"""Path to the PUDL ID mapping sheet with the plant map."""
 UTIL_ID_PUDL_MAP_CSV = importlib.resources.open_text(
     "pudl.package_data.glue", "utility_id_pudl.csv"
 )
-
+""
```

Comment:
Was this intended to be a docstring?

I think it would be clearer which constant the docstrings are attached to if there were a blank line between them.